src/interface/meta.thrift

```diff
@@ -608,7 +608,6 @@ struct GetConfigResp {
 
 struct SetConfigReq {
     1: ConfigItem           item,
-    2: bool                 force,
```

Comment:
We'd better don't remove the filed in thrift structure.  I

Reply:
Make sense.